R/pkg/R/sparkR.R

```diff
@@ -269,7 +269,7 @@ sparkR.sparkContext <- function(
 #' sparkR.session("yarn-client", "SparkR", "/home/spark",
 #'                list(spark.executor.memory="4g"),
 #'                c("one.jar", "two.jar", "three.jar"),
-#'                c("com.databricks:spark-avro_2.11:2.0.1"))
+#'                c("com.databricks:spark-avro_2.12:2.0.1"))
```

Comment:
@felixcheung is it OK to refer to _2.12 artifacts here? I don't think this one actually exists, but is it just an example?

Reply:
I think there's even a separate discussion about even using this as an example, since that package is now in Spark since 2.4.

Reply:
@felixcheung was the conclusion that we can make this a dummy package? I just want to avoid showing _2.11 usage here.

Reply:
yes, dummy name is completely fine with me.